src/FilterBundle/DependencyInjection/BestItCommercetoolsFilterExtension.php

```diff
@@ -32,6 +32,9 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setAlias('best_it_commercetools_filter.request.client', $config['client_id']);
         $container->setAlias('best_it_commercetools_filter.generator.url', $config['url_generator_id']);
 
+        // Set parameters
+        $container->setParameter('best_it_commercetools_filter.cache_life_time', $config['cache_life_time']);
```

Comment:
Naturally. Other projects may expect the data to come live. If now a cache is inserted, the behavior is changed. The data is no longer live and X hours old. In addition, the user has no way to deactivate the cache to restore the "old" behavior. So a breaking change without active / deactivate function in my opinion.